src/test/kotlin/com/karumi/maxibonkata/DevelopersTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.karumi.maxibonkata
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.properties.forAll
+import io.kotlintest.specs.ShouldSpec
+
+class DevelopersTest : ShouldSpec() {
+    init {
+
+        "Developers" {
+            should("always grab a positive number of maxibons") {
+                forAll({ numberOfMaxibons: Int ->
+                    val dev = Developer(numberOfMaxibons = numberOfMaxibons)
+
+                    dev.maxibonsToGrab >= 0
+                })
+            }
+
+            should("assign the name of the developer in construction") {
+                forAll({ name: String ->
+                    val dev = Developer(name)
+
+                    dev.name == name
+                })
+            }
```

Comment:
I know this test is not needed at all. It can also be considered a bad practice. But we use it during our training to show how to use ``forAll`` methods 😃